<launch>
  <node name="visual_cone_detector" pkg="iarrc_software" type="visual_cone_detector" output="screen">
        <param name="img_topic" type="string" value="/ps3_eye/image_color" />
  </node>
</launch>
